chia/server/server.py

```diff
@@ -267,13 +267,19 @@ async def start_server(self, on_connect: Callable = None):
         # this port from the socket itself and update self._port.
         self.site = web.TCPSite(
             self.runner,
-            host="0.0.0.0",
+            host="",  # should listen to both IPv4 and IPv6 on a dual-stack system
```

Comment:
@arvidn, this line is the fix for the production issue whereby you can not make IPv6 connections.  Fixing that causes tests to fail so the rest is needed to make the tests ok.

Reply:
right, I forgot about this when I made my comments. I just remembered the comment..

Reply:
yea, this one line change is enough for production, because the rest of the problems are actually limited to CI (we don't support using port=0 in production) - so the rest of the code is just a workaround around the fact that `port=0` ends up listening to multiple ports in CI when used in combination with `host=""`